superset-frontend/src/preamble.ts

```diff
@@ -74,3 +75,8 @@ setupColors(extraCategoricalColorSchemes, extraSequentialColorSchemes);
 
 // Setup number formatters
 setupFormatters();
+
+export const customTheme = merge(
+  supersetTheme,
+  bootstrapData?.common?.custom_theme ?? {},
+);
```

Comment:
Since this is merging into `supersetTheme`, it's not really the `custom_theme`, but rather a combination of both. Could this just be `theme`?

Reply:
we did in order to not enforce users override entire theme, so he can override just some props and other will be taken by default, because in other case some code can throw error that can't find some deep property in theme...

Reply:
Yes I'm not proposing not merging, but rather renaming it, as without any overrides it's not really a custom theme, but rather just the default theme. So the way I see it `theme = supersetTheme + themeOverrides` (pseudocode)

Reply:
ah yes did it already according second proposal